The deploy pipeline is failing to build the image

The build is failing with the following error message due to missing libXss

[ 2/11] RUN dnf update -y &&     dnf install -y         wget         curl         ca-certificates         findutils         binutils         tar         gzip         unzip         xorg-x11-server-Xvfb         nss         atk         gtk3         cups-libs         dbus-glib         libdrm         libXcomposite         libXdamage         libXrandr         libxkbcommon         libXScrnSaver         libXss         alsa-lib         fontconfig         liberation-fonts         mesa-libgbm         libgtk-3-0         libnss3         libxrandr2         libdrm2         libxkbcommon0         libxcomposite1         libxdamage1         libxss1         libasound2 &&     dnf clean all &&     rm -rf /var/cache/dnf:
1.421 Downloading metadata...
265.5 Nothing to do.
269.6 error: No package matches 'libXss'
------
Dockerfile:14
--------------------
  13 |     # Install comprehensive system dependencies for Playwright
  14 | >>> RUN dnf update -y && \
  15 | >>>     dnf install -y \
  16 | >>>         wget \
  17 | >>>         curl \
  18 | >>>         ca-certificates \
  19 | >>>         findutils \
  20 | >>>         binutils \
  21 | >>>         tar \
  22 | >>>         gzip \
  23 | >>>         unzip \
  24 | >>>         # Core X11 and graphics libraries
  25 | >>>         xorg-x11-server-Xvfb \
  26 | >>>         nss \
  27 | >>>         atk \
  28 | >>>         gtk3 \
  29 | >>>         cups-libs \
  30 | >>>         dbus-glib \
  31 | >>>         libdrm \
  32 | >>>         libXcomposite \
  33 | >>>         libXdamage \
  34 | >>>         libXrandr \
  35 | >>>         libxkbcommon \
  36 | >>>         libXScrnSaver \
  37 | >>>         libXss \
  38 | >>>         # Audio support
  39 | >>>         alsa-lib \
  40 | >>>         # Font support
  41 | >>>         fontconfig \
  42 | >>>         liberation-fonts \
  43 | >>>         # Additional dependencies for browser automation
  44 | >>>         mesa-libgbm \
  45 | >>>         libgtk-3-0 \
  46 | >>>         libnss3 \
  47 | >>>         libxrandr2 \
  48 | >>>         libdrm2 \
  49 | >>>         libxkbcommon0 \
  50 | >>>         libxcomposite1 \
  51 | >>>         libxdamage1 \
  52 | >>>         libxss1 \
  53 | >>>         libasound2 && \
  54 | >>>     dnf clean all && \
  55 | >>>     rm -rf /var/cache/dnf
  56 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c dnf update -y &&     dnf install -y         wget         curl         ca-certificates         findutils         binutils         tar         gzip         unzip         xorg-x11-server-Xvfb         nss         atk         gtk3         cups-libs         dbus-glib         libdrm         libXcomposite         libXdamage         libXrandr         libxkbcommon         libXScrnSaver         libXss         alsa-lib         fontconfig         liberation-fonts         mesa-libgbm         libgtk-3-0         libnss3         libxrandr2         libdrm2         libxkbcommon0         libxcomposite1         libxdamage1         libxss1         libasound2 &&     dnf clean all &&     rm -rf /var/cache/dnf" did not complete successfully: exit code: 1
UnfurlService-us-east-2: fail: docker build --build-arg DOCKER_BUILDKIT=1 --build-arg BUILDKIT_INLINE_CACHE=1 --tag cdkasset-15457552aad208078f71751d1de552eb59415d71210a73826c121b649b184311 --platform linux/arm64 . exited with error code 1: #0 building with "default" instance using docker driver

#1 [internal] load build definition from Dockerfile
#1 transferring dockerfile: 5.63kB done
#1 DONE 0.0s

#2 [internal] load metadata for public.ecr.aws/lambda/python:3.12-arm64
#2 DONE 0.4s

#3 [internal] load .dockerignore
#3 transferring context: 895B done
#3 DONE 0.0s

#4 [internal] load build context
#4 transferring context: 218.55kB done
#4 DONE 0.0s

#5 [ 1/11] FROM public.ecr.aws/lambda/python:3.12-arm64@sha256:d7f3bc91f565e9e3099c5ef50ddbbfe581a7fb0969b309f4639a953d38cdcc3c
#5 resolve public.ecr.aws/lambda/python:3.12-arm64@sha256:d7f3bc91f565e9e3099c5ef50ddbbfe581a7fb0969b309f4639a953d38cdcc3c done
#5 sha256:e480409eab85982ab02e2079325764366482d1ab21bf7b31fcfbb5c249ad39d3 0B / 416B 0.1s
#5 sha256:d7f3bc91f565e9e3099c5ef50ddbbfe581a7fb0969b309f4639a953d38cdcc3c 1.58kB / 1.58kB done
#5 sha256:43e7cd4a843b9712d67f42dbb684b548cac782fe40b54e94417e0947f78a80da 4.50kB / 4.50kB done
#5 sha256:34328c8be51cbad703d9e17aaa048fa432339731a4eccf6e25a78162a439aa8f 0B / 37.90MB 0.1s
#5 sha256:287352861cd86eefd1[307](https://github.com/willtech3/unfurl-service/actions/runs/15441142418/job/43459087750#step:12:308)2fb24ca6451e09f345151080dbdc3bcd3f302abcf3f 0B / 88.13kB 0.1s
#5 sha256:e480409eab85982ab02e2079325764366482d1ab21bf7b31fcfbb5c249ad39d3 416B / 416B 0.1s done
#5 sha256:34328c8be51cbad703d9e17aaa048fa432339731a4eccf6e25a78162a439aa8f 37.90MB / 37.90MB 0.3s done
#5 sha256:287352861cd86eefd13072fb24ca6451e09f345151080dbdc3bcd3f302abcf3f 88.13kB / 88.13kB 0.1s done
#5 sha256:dc7934710945a440aa997dbdb1c84f02a15959ce7c025758ca7d34d40de25d7b 25.17MB / 143.35MB 0.3s
#5 sha256:ef05d9f2265b34e7a00e7aff623707cf81b4e793a99bdeb46a7b38d0974e944b 2.72MB / 2.72MB 0.3s done
#5 sha256:7f6217391417ac5cd3e3b0a793f67325f395bb5542d3546ce6e801c9a47c86c3 0B / 14.19kB 0.3s
#5 sha256:dc7934710945a440aa997dbdb1c84f02a15959ce7c025758ca7d34d40de25d7b 54.67MB / 143.35MB 0.4s
#5 sha256:7f6217391417ac5cd3e3b0a793f67325f395bb5542d3546ce6e801c9a47c86c3 14.19kB / 14.19kB 0.3s done
#5 extracting sha256:34328c8be51cbad703d9e17aaa048fa4[323](https://github.com/willtech3/unfurl-service/actions/runs/15441142418/job/43459087750#step:12:324)39731a4eccf6e25a78162a439aa8f
#5 sha256:dc7934710945a440aa997dbdb1c84f02a15959ce7c025758ca7d34d40de25d7b 120.60MB / 143.35MB 0.6s
#5 sha256:dc7934710945a440aa997dbdb1c84f02a15959ce7c025758ca7d34d40de25d7b 143.35MB / 143.35MB 0.7s
#5 sha256:dc7934710945a440aa997dbdb1c84f02a15959ce7c025758ca7d34d40de25d7b 143.35MB / 143.35MB 0.8s done
#5 extracting sha256:34328c8be51cbad703d9e17aaa048fa432339731a4eccf6e25a78162a439aa8f 1.9s done
#5 extracting sha256:287352861cd86eefd13072fb24ca6451e09f345151080dbdc3bcd3f302abcf3f done
#5 extracting sha256:e480409eab85982ab02e2079[325](https://github.com/willtech3/unfurl-service/actions/runs/15441142418/job/43459087750#step:12:326)764366482d1ab21bf7b31fcfbb5c249ad39d3
#5 extracting sha256:e480409eab85982ab02e2079325764366482d1ab21bf7b31fcfbb5c249ad39d3 done
#5 extracting sha256:ef05d9f2265b34e7a00e7aff623707cf81b4e793a99bdeb46a7b38d0974e944b 0.0s done
#5 extracting sha256:dc7934710945a440aa997dbdb1c84f02a15959ce7c025758ca7d34d40de25d7b
#5 extracting sha256:dc7934710945a440aa997dbdb1c84f02a15959ce7c025758ca7d34d40de25d7b 4.0s done
#5 extracting sha256:7f6217391417ac5cd3e3b0a793f67325f395bb5542d3546ce6e801c9a47c86c3
#5 extracting sha256:7f6217391417ac5cd3e3b0a793f67325f395bb5542d3546ce6e801c9a47c86c3 done
#5 DONE 7.5s

#6 [ 2/11] RUN dnf update -y &&     dnf install -y         wget         curl         ca-certificates         findutils         binutils         tar         gzip         unzip         xorg-x11-server-Xvfb         nss         atk         gtk3         cups-libs         dbus-glib         libdrm         libXcomposite         libXdamage         libXrandr         libxkbcommon         libXScrnSaver         libXss         alsa-lib         fontconfig         liberation-fonts         mesa-libgbm         libgtk-3-0         libnss3         libxrandr2         libdrm2         libxkbcommon0         libxcomposite1         libxdamage1         libxss1         libasound2 &&     dnf clean all &&     rm -rf /var/cache/dnf
#6 1.421 Downloading metadata...
#6 265.5 Nothing to do.
#6 269.6 error: No package matches 'libXss'
#6 ERROR: process "/bin/sh -c dnf update -y &&     dnf install -y         wget         curl         ca-certificates         findutils         binutils         tar         gzip         unzip         xorg-x11-server-Xvfb         nss         atk         gtk3         cups-libs         dbus-glib         libdrm         libXcomposite         libXdamage         libXrandr         libxkbcommon         libXScrnSaver         libXss         alsa-lib         fontconfig         liberation-fonts         mesa-libgbm         libgtk-3-0         libnss3         libxrandr2         libdrm2         libxkbcommon0         libxcomposite1         libxdamage1         libxss1         libasound2 &&     dnf clean all &&     rm -rf /var/cache/dnf" did not complete successfully: exit code: 1
------
 > [ 2/11] RUN dnf update -y &&     dnf install -y         wget         curl         ca-certificates         findutils         binutils         tar         gzip         unzip         xorg-x11-server-Xvfb         nss         atk         gtk3         cups-libs         dbus-glib         libdrm         libXcomposite         libXdamage         libXrandr         libxkbcommon         libXScrnSaver         libXss         alsa-lib         fontconfig         liberation-fonts         mesa-libgbm         libgtk-3-0         libnss3         libxrandr2         libdrm2         libxkbcommon0         libxcomposite1         libxdamage1         libxss1         libasound2 &&     dnf clean all &&     rm -rf /var/cache/dnf:
1.421 Downloading metadata...
265.5 Nothing to do.
269.6 error: No package matches 'libXss'
------
Dockerfile:14
--------------------
  13 |     # Install comprehensive system dependencies for Playwright
  14 | >>> RUN dnf update -y && \
  15 | >>>     dnf install -y \
  16 | >>>         wget \
  17 | >>>         curl \
  18 | >>>         ca-certificates \
  19 | >>>         findutils \
  20 | >>>         binutils \
  21 | >>>         tar \
  22 | >>>         gzip \
  23 | >>>         unzip \
  24 | >>>         # Core X11 and graphics libraries
  25 | >>>         xorg-x11-server-Xvfb \
  26 | >>>         nss \
  27 | >>>         atk \
  28 | >>>         gtk3 \
  29 | >>>         cups-libs \
  30 | >>>         dbus-glib \
  31 | >>>         libdrm \
  32 | >>>         libXcomposite \
  33 | >>>         libXdamage \
  34 | >>>         libXrandr \
  35 | >>>         libxkbcommon \
  36 | >>>         libXScrnSaver \
  37 | >>>         libXss \
  38 | >>>         # Audio support
  39 | >>>         alsa-lib \
  40 | >>>         # Font support
  41 | >>>         fontconfig \
  42 | >>>         liberation-fonts \
  43 | >>>         # Additional dependencies for browser automation
  44 | >>>         mesa-libgbm \
  45 | >>>         libgtk-3-0 \
  46 | >>>         libnss3 \
  47 | >>>         libxrandr2 \
  48 | >>>         libdrm2 \
  49 | >>>         libxkbcommon0 \
  50 | >>>         libxcomposite1 \
  51 | >>>         libxdamage1 \
  52 | >>>         libxss1 \
  53 | >>>         libasound2 && \
  54 | >>>     dnf clean all && \
  55 | >>>     rm -rf /var/cache/dnf
  56 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c dnf update -y &&     dnf install -y         wget         curl         ca-certificates         findutils         binutils         tar         gzip         unzip         xorg-x11-server-Xvfb         nss         atk         gtk3         cups-libs         dbus-glib         libdrm         libXcomposite         libXdamage         libXrandr         libxkbcommon         libXScrnSaver         libXss         alsa-lib         fontconfig         liberation-fonts         mesa-libgbm         libgtk-3-0         libnss3         libxrandr2         libdrm2         libxkbcommon0         libxcomposite1         libxdamage1         libxss1         libasound2 &&     dnf clean all &&     rm -rf /var/cache/dnf" did not complete successfully: exit code: 1
Failed to build asset UnfurlProcessor/AssetImage
Error: Process completed with exit code 1.

--- a/Dockerfile
+++ b/Dockerfile
@@ -34,23 +34,13 @@ RUN dnf update -y && \
         libXrandr \
         libxkbcommon \
         libXScrnSaver \
-        libXss \
         # Audio support
         alsa-lib \
         # Font support
         fontconfig \
         liberation-fonts \
         # Additional dependencies for browser automation
-        mesa-libgbm \
-        libgtk-3-0 \
-        libnss3 \
-        libxrandr2 \
-        libdrm2 \
-        libxkbcommon0 \
-        libxcomposite1 \
-        libxdamage1 \
-        libxss1 \
-        libasound2 && \
+        mesa-libgbm && \
     dnf clean all && \
     rm -rf /var/cache/dnf
 

